### DISCUSSION TITLE
come salvare enne tabelle

### DISCUSSION THREAD

**Discussion by pigreco** [ondata/guidaVisiData#26] [Domande] [answered]
con questo comando scarico tutte le tabelle dal sito:

```
vd https://lab24.ilsole24ore.com/ecosistema-urbano/indexT.php
```
ottenendo questo:

![image](https://user-images.githubusercontent.com/7631137/141001955-5275a627-065f-4188-af0a-7d74c4646462.png)

ogni riga della tabella di sopra è a sua volta una tabella, come salvare in locale tutte le tabelle collegate?

![image](https://user-images.githubusercontent.com/7631137/141002006-989b3189-8982-41d3-afd5-1e73979ae9e6.png)

**Comment by aborruso** [comment, marked as answer]
Caro @pigreco , rispondo adesso un po' di fretta.

In questi casi, per salvare in blocco in un formato, bisogna fissare in apertura qual è il formato in cui si salverà in blocco:

```
vd https://lab24.ilsole24ore.com/ecosistema-urbano/indexT.php --save-filetype=csv
```

Una volta fatto:

- rimanere nelle prima colonna e selezionare tutto `gs`;
- per evitare nomi tabella tutti uguale digitare `gi`;
- lanciare il comando salva tutto `g+CTRL+s`;
- indicare come output la cartella corrente `./` (occhio quindi, verificare che non ci sia nulla di importante dentro) e dare INVIO;
- chiederà permesso di sovrascrivere in ogni caso (dare yes).

Appena puoi prova e fammi sapere

**Comment by pigreco** [comment]
@aborruso 
ho appena fatto un test,
semplicemente fantastico, grazie

https://user-images.githubusercontent.com/7631137/141086711-62d56e19-5aa0-4605-adaf-18455744d148.mp4

**Reply by aborruso** [reply]
È stata inserita nella guida https://ondata.github.io/guidaVisiData/ricette/?h=tabelle#salvare-una-tabella-html-in-csv-a-partire-da-una-pagina-web
